src/httpserver.cpp

```diff
@@ -21,19 +21,22 @@
 #include <util/threadnames.h>
 #include <util/translation.h>
 
+#include <condition_variable>
 #include <cstdio>
 #include <cstdlib>
 #include <deque>
 #include <memory>
 #include <optional>
 #include <string>
+#include <unordered_set>
 
 #include <sys/types.h>
 #include <sys/stat.h>
 
 #include <event2/buffer.h>
 #include <event2/bufferevent.h>
 #include <event2/http.h>
+#include <event2/http_struct.h>
```

Comment:
nit: IWYU suggests to s/`#include <event2/http_struct.h>`/`#include <event2/event.h>`/.

Reply:
Thanks! I will fix this in the follow-up that will pick up #19434.